src/reducers/computeRenderedRows.js

```diff
@@ -145,6 +145,23 @@ function calculateRenderedRowRange(state, scrollAnchor) {
     rowIdx += step;
   }
 
+  /* Handle the case where we rows have shrunk and we don't have enough content
+     between our start scroll anchor and the end of the table to fill the available space.
+     In this case process earlier rows as needed and act as if we've scrolled to the last row.
+   */
+  let forceScrollToLastRow = false;
+  if (totalHeight < maxAvailableHeight && rowIdx === rowsCount && lastIndex === undefined) {
+    forceScrollToLastRow = true;
+    const reverseStep = -1;
+    rowIdx = firstIndex - 1;
+
+    while (rowIdx >= 0 && totalHeight < maxAvailableHeight) {
+      totalHeight += updateRowHeight(state, rowIdx);
+      startIdx = rowIdx;
+      rowIdx += reverseStep;
```

Comment:
Unless there's intention to take bigger steps in the future, can we remove `reverseStep`, and then `--rowIdx;` here?

Reply:
Yep, that's fine.  Will do

Reply:
done